tools/codegen/pkg/proc_interface_gen/generator.go

```diff
@@ -0,0 +1,97 @@
+// Copyright 2017 Istio Authors
```

Comment:
i feel like this file should be in a directory that matches the package name: `interfacegen`